{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/config.py

```diff
@@ -26,16 +26,13 @@ def _(x):
         search_index='records',
         search_type=None,
         record_serializers={
-            'application/json': ('{{ cookiecutter.package_name }}.records.serializers'
-                                 ':json_v1_response'),
+            'application/json': ('{{ cookiecutter.package_name }}.records.serializers:json_v1_response'),
```

Comment:
This is likely to generate too long names when a user, uses something else than ``my-site``.